File: Desenvolvimento/class/disco.php
<?php

class disco{
    private $cod_disco;
    private $marca;
    private $modelo;
    private $espessura;
    private $diametro;
    private $tempo_trabalho;

    public function setCod_disco($cod_disco){
        $this->cod_disco = $cod_disco;
    }
    public function getCod_disco(){
        return $this->cod_disco;
    }
    public function setMarca($marca){
        $this->marca = $marca;
    }
    public function getMarca(){
        return $this->marca;
    }
    public function setModelo($modelo){
        $this->modelo = $modelo;
    }
    public function getModelo(){
        return $this->modelo;
    }
    public function setEspessura($espessura){
        $this->espessura = $espessura;
    }
    public function getEspessura(){
        return $this->espessura;
    }
    public function setDiametro($diamentro){
        $this->diametro = $diamentro;
    }
    public function getDiamentro(){
        return $this->diametro;
    }
    public function setTempo_trabalho($tempo_trabalho){
        $this->tempo_trabalho = $tempo_trabalho;
    }
    public function getTempo_trabalho(){
        return $this->tempo_trabalho;
    }
}

?>

File: Sistema de Corte de Rochas - Caso de Uso/src/javafx/domain/Disco.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package javafx.domain;

/**
 *
 * @author André
 */
public class Disco {

    private int codDisco;
    private String marca;
    private String modelo;
    private int espessura;
    private int diametro;
    private Double tempoTrabalho;

    public int getCodDisco() {
        return codDisco;
    }

    public void setCodDisco(int codDisco) {
        this.codDisco = codDisco;
    }

    public String getMarca() {
        return marca;
    }

    public void setMarca(String marca) {
        this.marca = marca;
    }

    public String getModelo() {
        return modelo;
    }

    public void setModelo(String modelo) {
        this.modelo = modelo;
    }

    public int getEspessura() {
        return espessura;
    }

    public void setEspessura(int espessura) {
        this.espessura = espessura;
    }

    public int getDiametro() {
        return diametro;
    }

    public void setDiametro(int diametro) {
        this.diametro = diametro;
    }

    public Double getTempoTrabalho() {
        return tempoTrabalho;
    }

    public void setTempoTrabalho(Double tempoTrabalho) {
        this.tempoTrabalho = tempoTrabalho;
    }

    @Override
    public String toString() {
        return this.marca+" - "+this.modelo;
    }

}
